python/ray/tune/scripts.py

```diff
@@ -35,10 +42,17 @@ def list_trials(experiment_path, sort, output):
     "-o",
     default=None,
     type=str,
-    help="Select filename to output information to.")
-def list_experiments(project_path, sort, output):
+    help="Select file to output information to.")
+@click.option(
+    "--filter",
+    "--filter_op",
+    nargs=3,
```

Comment:
Make this a single argument, such that the filter is always a single string that you parse.